packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.component.tsx

```diff
@@ -13,31 +13,37 @@ import {
   Grid,
   InlineNotification,
   TextInput,
-  FormLabel,
+  TextArea,
 } from '@carbon/react';
 import { useTranslation } from 'react-i18next';
 import { priorityOptions } from './lab-order';
 import { useTestTypes } from './useTestTypes';
 import styles from './lab-order-form.scss';
-import { Controller, useForm } from 'react-hook-form';
+import { Controller, type FieldErrors, useForm } from 'react-hook-form';
 import { zodResolver } from '@hookform/resolvers/zod';
 import { z } from 'zod';
+import { moduleName } from '@openmrs/esm-patient-chart-app/src/constants';
 
 export interface LabOrderFormProps {
   initialOrder: LabOrderBasketItem;
   closeWorkspace: () => void;
 }
 
 const labOrderFormSchema = z.object({
-  urgency: z.string({ required_error: 'Priority is required', invalid_type_error: 'Priority is required' }),
+  urgency: z.string({
+    required_error: translateFrom(moduleName, 'addLabOrderPriorityRequired', 'Priority is required'),
+    invalid_type_error: translateFrom(moduleName, 'addLabOrderPriorityRequired', 'Priority is required'),
```

Comment:
@vasharma05, wasn't the idea to remove `invalid_type_error` here?

Reply:
I have remove this here